forge-gui/res/draft/rankings/dsk.rnk

```diff
@@ -91,7 +91,7 @@
 #90|Cursed Windbreaker|U|DSK
 #91|Insidious Fungus|U|DSK
 #92|Paranormal Analyst|U|DSK
-#93|Unholy Heat|M|DSK
+#93|Unholy Heat|M|SPG
```

Comment:
Is this right?

Reply:
https://scryfall.com/card/spg/71/unholy-heat

Yeah a lot of the SPG and alternative sets didn't have a ranking

Reply:
Ah yea, this is the thing i needed to fiddle with in different ranking files.